actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -343,3 +343,58 @@ fn effective_gas_limit<RT: Runtime>(system: &System<RT>, gas: U256) -> u64 {
     let gas = gas.to_u64_saturating();
     std::cmp::min(gas, gas_rsvp)
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Can we test out-of-bounds behavior?

Reply:
Sure, but is the specified behaviour we want to assert?

I dont want to just certify what the code does right now, it is vacuous (and could well be wrong)

Reply:
- CALLDATALOAD always works, no matter how big/small the input data is.
- CALLDATACOPY automatically treats out-of-bounds reads as "zeros".

Reply:
done